### ISSUE TITLE
MemoryError

### ISSUE THREAD

**Issue by FilimonovAP81** [latchset/jwcrypto#110]
Hello,

I have MemoryError message then I try to import your lib.

Do you know potential problem places? 

I use python 2.7.10 and djangorestframework 3.3.2 and last release of your library.
Result is:
```
Exception Type: |MemoryError
/usr/lib64/python2.7/ctypes/__init__.py in _reset_cache, line 279
```

**Comment by simo5** [comment]
@tiran could you take a look at this one ?

**Comment by tiran** [comment]
Memory error means that your machine has no free memory. Please tell us more about your setup. Which Python version, OS, vendor and CPU architecture are you running and how much memory does your machine have?

**Comment by FilimonovAP81** [comment]
No, this is not a really memory error.
I have already fixed it. The problem in cryptography library. As I understand this correctly, same of them wrote on C. And problem in the /usr/lib64/python2.7/ctypes/__init__.py in _reset_cache, line 279 append then python try to work with C libs.  May be I am mistake but any way I uninstalled cryptography library and all works fine now.

Thank you for your help.
